### PR TITLE
Updated README with with prerequisites on deploying siddhi apps

### PR DESCRIPTION
**Pull request by Ambujaa10**
### Proposed changes in this pull request

[List all changes you want to add here. If you fixed an issue, please
add a reference to that issue as well.]

- README.md is altered in order to provide the set of instructions to deploy siddhi4 file.

### When should this PR be merged

[Please describe any preconditions that need to be addressed before we
can merge this pull request.]
N/A

### Follow up actions

[List any possible follow-up actions here; for instance, testing data
migrations, software that we need to install on staging and production
environments.]

-

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description. N?A
- [ ] **Is the PR labeled correctly?** yes

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification. N/A
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. N/A

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way. N/A
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. N/A
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). N/A

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components.
- [ ] **If this is a bug fix, are tests for the issue in place?**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature?** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?**
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes should be documented.
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented.
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

File: features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md [hunk 1]
```diff
@@ -1 +1,6 @@
+In order to build the Siddhi4 execution files
```
Comment (Niveathika92) [inline features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md]: Is this a prerequisite? To build siddhi apps why is the repository need to be cloned?

**Review by niveathika — CHANGES_REQUESTED** [review]

File: features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md (hunk 1 above)
Comment (Ambujaa10) [reply features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md]: Ack. thank you

**Comment by Ambujaa10** [comment]
Changes ON README.md

**Comment by Niveathika92** [comment]
It's better to give more context in the PR heading as well as the description. Something like, "updated README with prerequisites on deploying siddhi apps"? It applies to commit messages as well.

**Comment by Ambujaa10** [comment]
ok akka thank you.
I will make the next PR as you said

**Comment by Ambujaa10** [comment]
Ack. thank you for sharing

**Comment by Ambujaa10** [comment]
Custom widget for SP dashboard is created. It's a line chart which displays the variation of session count with time.

**Review by ruwanta — APPROVED** [review]

**Pull request merged**